core/sets.gms

```diff
@@ -1325,6 +1328,7 @@ teNoRe(all_te)        "Non renewable energy technologies"
 teVRE(all_te)      "technologies requiring storage"
 /
         wind        "wind power converters"
+        windoff     "wind offshore power converters"
```

Comment:
wind = windon, I didn't want to make all the necessary name changes through out the code, I asked the group if they were ok with this in a REMIND meeting some weeks ago, no one objected

I can implement gridwindoff and storwindoff as well (I can ask Robert)

The idea was that before: wind = windon +windoff, now wind = wind on, and windoff = wind off. Before it is aggregated, now we would like to disaggregate

Reply:
If "wind" is being changed to refer only to "windon", I would prefer if the name is also changed to avoid confusion.

Did you changed the remind2 R library to consider windoff now whenever it only considered wind? 

Did you split the the historical capacities for wind in mrremind to represent now the split between wind (windon) and windoff?
Also, did you adjusted the wind core/bounds.gms accordingly?
As you are changing the previous wind set name from total wind to windon only, you need to adjust the historical bounds and data to be compatible also, if not you are not representing correctly any region that already have installed off-shore capacity.

Reply:
Thanks Renato, technically I can change the name from wind to windon, however, I will have to change the remind2 library (and remind library as well) so it is all compatible (probably as well as all the input libraries, mrremind, madrat, etc, it might be too much hassle and might crash in places I haven't thought of...) It sounds like Gunnar and our ARIADNE partners want a quick solution for Germany, so I think we can change this later

if you mean if remind2 library is backward compatible, yes it is. (so a gdx without winoff in it will be reported without bugs)

Reply:
If you don't change the R libraries you are creating a mismatch between the model data input and the reported results.

Anything related with wind off-shore will just be present in the gams part, and will be missing from any reported variables, or not considered when defining wind on shore capacities.

Reply:
For a quick solution, you could simply calculate that after the model run in the reporting, but if you are changing the wind set definition, you are change input data and reporting variables consequently.